packages/as-transformer/src/transformer.ts

```diff
@@ -85,7 +64,13 @@ export class Transformer extends TransformVisitor {
     const inputText = (node.expression as IdentifierExpression)?.text;
 
     for (let transformer of callTransformers) {
+      if (
+        (transformer as MassaExportCalls) !== undefined &&
+        this.originalVisiting
+      )
```

Comment:
Can you add a comment explaining why you continue here?

Reply:
I don't see any explanaition. Did I miss something?

Reply:
That's odd, on my side I pushed changes.

Reply:
You don't see them @gregLibert ?